docker/bin/entrypoint.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+set -Eeo pipefail
+
+if [[ $1 == *"bash"* || $1 == *"sh"* ]]; then
+    : # noop
+elif [[ "$*" == *"--help"* || "$*" == *"list "* ]]; then
+    set -- esrally "$@" # allow --help, --list commands to be passed as is
```

Comment:
Did you mean to write `list` instead of `--list`?  Also, why does this need special treatment? What about other subcommands like `compare`?

Reply:
Good catch left over from previous logic. We don't need special handling for parameters any more, only the "syntactic sugar" of prefixing `esrally` if not specified. Will address.